server/start.go

```diff
@@ -68,9 +68,26 @@ import (
 	ethermint "github.com/evmos/ethermint/types"
 )
 
+type DBOpener func(types.AppOptions, string, dbm.BackendType) (dbm.DB, error)
```

Comment:
nit, comment and add names for the variables

Reply:
done